crates/rome_css_parser/README.md

```diff
@@ -1,36 +1,23 @@
 <p align="center">
-	<picture>
-		<source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/rome/brand/main/PNG/logo_white_yellow_transparent.png" width="700">
-		<img alt="Rome's logo depicting an ancient Roman arch with the word Rome to its side" src="https://raw.githubusercontent.com/rome/brand/main/PNG/logo_transparent.png" width="700">
-	</picture>
+	<img alt="Biome - Toolchain of the web" width="400" src="https://raw.githubusercontent.com/biomejs/resources/main/biome-logo-slogan.svg"/>
```

Comment:
Don't we have a light/dark version?

Reply:
I think the current logo is a good fit for both theme?